docs/_templates/2023/index.html

```diff
@@ -0,0 +1,195 @@
+{% extends year_str + "/base.html" %}
+
+{% block full_title %}
+Home - Write the Docs {{ name }} {%- if cobrand is defined and cobrand %} + {{ cobrand.brand }} {% endif %} {{ year }}
+{% endblock %}
+
+{% block head_link %}
+<style>
+.page-content h1 {
+    display: none;
+}
+</style>
+{% endblock %}
+
+{% block container %}
+<div class="hero" style="background: linear-gradient(rgba({{ rgb }}, 1),  rgba({{ rgb }}, 0.5)), url({{ pathto(meta['banner'], 1) }}), no-repeat;background-position: bottom;background-size: cover;">
+    <nav class="uk-navbar-container uk-container uk-navbar-transparent" uk-navbar="offset: -18">
+      <div class="uk-navbar-right">
+        <button class="uk-navbar-toggle uk-hidden@s"  type="button" uk-navbar-toggle-icon uk-toggle="target: #offcanvas-nav"></button>
+        <div id="offcanvas-nav" uk-offcanvas="overlay: true">
+            <div class="uk-offcanvas-bar uk-flex uk-flex-column">
+                    <div class="menu-logo">
+                            Write<br>
+                            the<br>
+                            Docs<br>
+                            {% if cobrand is defined and cobrand %}
+                            <em>+</em>
+                            {{ cobrand.brand_html }}
+                            {% endif %}
+                            <em>{{ name }}</em> <br>
+                            <em style="top: -20px;">{{ year }}</em>
+                        </div>
+                      {% if not flaglanding %}
+                     {% include year_str + "/menu-mobile.html" %}
+                     {% endif %}
+            </div>
+        </div>
+       {% if not flaglanding %}
+       {% include year_str + "/menu-desktop.html" %}
+       {% endif %}
+      </div>
+    </nav>
+
+    <div class="hero-content uk-container">
+      <div class="logo-container">
+        <a class="hero-logo" data-proofer-ignore>
+          Write<br/>
+          the<br/>
+          Docs<br/>
+          {% if cobrand is defined and cobrand %}
+          <em>+</em>
+          {{ cobrand.brand_html }}
+          {% endif %}
+          <em>{{ name }}</em> <br>
+          <em style="top: -20px;">{{ year_str }}</em>
+        </a>
+      </div>
+      <div class="hero-text">
+        <p>The conference focused on all things related to software documentation.</p>
+        <p>
+          <b>
+            Online: {{ tz }} <br>
+            <!--{{ city }}, {{ local_area }}<br> -->
```

Comment:
We need a conditional here, since this is showing for Portland still.

Reply:
@mxsasha I hacked this for now, but we probably want some kind of flag saying if a conf is virtual or not?